docs/reference/system/policies.md

```diff
@@ -36,15 +36,15 @@ If this policy grants the user admin access. This means that users with this pol
 `app_access` **boolean**\
 Whether or not users with this policy have access to use the Data Studio.
 
-`users` **one-to-many**\
-The users this policy is assigned to directly, this does not include users which receive this policy through a role. One-to-many
-to [users](/reference/system/users).
+`users` **many-to-many**\
+The users this policy is assigned to directly via `directus_access`, this does not include users which receive this policy
+through a role. Many-to-many to [access](/reference/system/access).
```

Comment:
It's still targeting `directus_users` though, `directus_access` is just the junction table. Right?

```suggestion
through a role. Many-to-many to [users](/reference/system/users).
```

Same with the `roles` field.